source/frontend/src/utils/download.ts

```diff
@@ -1,81 +0,0 @@
-import { AxiosRequestConfig } from 'axios';
-import CustomAxios from 'customAxios';
-import { hideLoading, showLoading } from 'react-redux-loading-bar';
-import { AnyAction } from 'redux';
-import { ThunkAction } from 'redux-thunk';
-import { logError, logRequest, logSuccess } from 'store/slices/network/networkSlice';
-import { RootState } from 'store/store';
-
-/**
- * Download configuration options interface.
- */
-export interface IDownloadConfig extends AxiosRequestConfig {
-  fileName: string;
-  actionType: string;
-}
-
-/**
- * Programmatically triggers a file download with content generated through an API
- * @param filename the file name
- * @param blobData Raw blob data as returned by a file export API
- */
-export const downloadFile = (filename: string, blobData: any) => {
```

Comment:
technically we were still using this but only for user management so I updated that code to follow our conventions and then was able to remove this.